src/components/CheckBoxTree/style.css

```diff
@@ -6,6 +6,10 @@
     flex-flow: row;
 }
 
+.sub-menu :global(.ant-col-offset-3) {
+    margin-left: 13.4%;
```

Comment:
was this just a trial and error value?

Reply:
Yes, it's overriding an existing percentage value